pull_data.R

```diff
@@ -0,0 +1,33 @@
+library(sbtools)
+library(data.table)
+library(readxl)
+
+source("auth.R")
```

Comment:
I don't have a problem with this as is, just wanted to clarify.  I think we may want to play around with other places to put these credentials, as we will probably run into this again.